change/@fluentui-react-toolbar-c5617b54-e748-466b-a99a-b8f279ba376d.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 16 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27796&commitId=7ff217b19971721daf9a9ab25d9bcb5349dd4824&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1917 screenshots unchanged, 0 screenshots with different dimensions and 16 screenshots with visible difference.


  <details><summary><b>unknown</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Toolbar Converged.Default.Button Pressed.chromium.png|5334| Changed|
|Toolbar Converged.Default.Toggle On.chromium.png|5279| Changed|
|Toolbar Converged.Default.default.chromium.png|5278| Changed|
|Toolbar Converged.Far Group.chromium.png|2618| Changed|
|Toolbar Converged.Large.Button Pressed.chromium.png|5936| Changed|
|Toolbar Converged.Large.Toggle On.chromium.png|5804| Changed|
|Toolbar Converged.Large.default.chromium.png|5805| Changed|
|Toolbar Converged.Small.Button Pressed.chromium.png|5340| Changed|
|Toolbar Converged.Small.Toggle On.chromium.png|5291| Changed|
|Toolbar Converged.Small.default.chromium.png|5286| Changed|
|Toolbar Converged.Vertical Button.Button Pressed.chromium.png|8761| Changed|
|Toolbar Converged.Vertical Button.Toggle On.chromium.png|8665| Changed|
|Toolbar Converged.Vertical Button.default.chromium.png|8685| Changed|
|Toolbar Converged.Vertical.Button Pressed.chromium.png|19553| Changed|
|Toolbar Converged.Vertical.Toggle On.chromium.png|19465| Changed|
|Toolbar Converged.Vertical.default.chromium.png|19490| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>